stats/worker.go

```diff
@@ -294,7 +294,12 @@ func (w *worker) reportUsage(now time.Time) {
 			e.Export(viewData)
 		}
 		exportersMu.Unlock()
-		if _, ok := v.Window().(*Cumulative); !ok {
+		switch v.Window().(type) {
+		case *Cumulative:
+			// noop
+		case Cumulative:
+			// noop
+		default:
 			v.clearRows()
```

Comment:
I don't think we want to call clearRows at all -- otherwise if we have a one-minute window, it will only contain the last 10 seconds because we keep clearing it.